StyleCop.Analyzers/StyleCop.Analyzers/DocumentationRules/SA1642SA1643CodeFixProvider.cs

```diff
@@ -53,13 +54,19 @@ public override FixAllProvider GetFixAllProvider()
                     continue;
                 }
 
-                var node = root.FindNode(diagnostic.Location.SourceSpan, findInsideTrivia: true) as XmlElementSyntax;
-                if (node == null)
+                var node = root.FindNode(diagnostic.Location.SourceSpan, findInsideTrivia: true) as XmlNodeSyntax;
```

Comment:
We've had some problems in the past where failing to specify `getInnermostNodeForTie: true` resulted in exceptions when running the code fix on documents with syntax errors. In addition, the `as XmlNodeSyntax` appears to be unnecessary. I recommend the following:

```csharp
SyntaxNode node = root.FindNode(diagnostic.Location.SourceSpan, findInsideTrivia: true, getInnermostNodeForTie: true);
```